include/decord/runtime/ndarray.h

```diff
@@ -35,7 +35,7 @@ namespace runtime {
 class NDArray {
  public:
   // pts of the frame
-  int pts=0;
+  int pts=-1;
```

Comment:
will `pts` be checked by `NextFrameImpl`?

Reply:
No, NextFrameImpl currently does not check this value. This value is checked in CheckKeyFrame to detect eof pattern.

BTW, how about using AV_NOPTS_VALUE constant as the init value of pts, instead of this -1?

Reply:
It's actually fine with -1, introducing AV_NOPTS_VALUE to runtime lib will add an extra dependency in the core modules.